README.md

```diff
@@ -460,6 +462,18 @@ To override any of the defaults, begin by creating the corresponding file (from
 `~/.config/bs/keymap.toml`.
 You can change keybindings configuration file by exporting the `KEYMAP_FILE` environment variable.
 
+## Integrations
+
+BugStalker has integration with some useful tools that can help you when researching programs.
+
+### Binsider
+
+According to [official site](https://binsider.dev/), `binsider` offers powerful static and dynamic analysis tools,
+similar to readelf(1) and strace(1).
+It lets you inspect strings, examine linked libraries, and perform hexdumps, all within a user-friendly TUI.
+
+- `binsider` - run binsider inside a debugger
```

Comment:
Consider also supporting a short version like `b` or `bn` since `binsider` might be a long to type.

Also, we can have more specific commands as shortcuts, for example:

- `static_info` -> `binsider --tab static-analysis`
- `strings` -> `binsider --tab strings`

And so on :)

Reply:
Good idea! I'm add `bn` shortcut. About specific commands - i think it's nice too, but looks like we need to make simple api for it first :)

Reply:
Sounds reasonable!